docs/content/en/docs/Development/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Development
+weight: 30
+---
+
+## Pre-Commit
+
+We use [pre-commit](https://pre-commit.com/) to manage a series of git
+pre-commit hooks for the project; for example, each time you commit code, the
+hooks will make sure that your C++ is formatted properly. If your code isn't,
+the hook will format it, so when you try to commit the second time you'll get
+past the hook. Note that spelling mistakes identified by the codespell hook will
+not be auto-corrected and require manual resolution, rather than simply
+re-running pre-commit.
+
+All hooks are defined in `.pre-commit-config.yaml`. To install these hooks, run
```

Comment:
to an inattentive reader, this might sound like `pip install -r ...` is all they need to do, so maybe add a `first`?
```suggestion
All hooks are defined in `.pre-commit-config.yaml`. To install these hooks, first run
```